features/types/types_adminstration.feature

```diff
@@ -90,4 +90,4 @@ Feature: Type Administration
   Scenario: A standard type cannot be deleted
      When I go to the global index page of types
      Then "Standard" should be the last element in the list
```

Comment:
Note: `first element`/`last element` steps are actually broken (they don't check position in the list). Should be fixed here and in other places, but out-of-scope for this PR probably.